### ISSUE TITLE
NBT Tag with "as byte[]"

### ISSUE THREAD

**Issue by DerOli82** [jaredlll08/CraftTweaker#83]
I don't know if it's a misunderstanding, a wrong usage or a bug.

if i try to use an byte array:
`[ 1, 2, 3 ] as byte[]`
I'll get this error:
`cannot cast byte[] to minetweaker.data.IData`

"Real World" Exampe:
```
recipes.addShaped( thermExpInductionsmelter.withTag({
  "RSControl": 0 as byte,
  "Facing": 3,
  "Energy": 0,
  "Level": 0 as byte,
  "SideCache": [3, 1, 2, 2, 2, 2] as byte[],
  "Augments": [
    { "Slot": 0, "id": 5627 , "Count": 1 , "Damage": 0 }, 
    { "Slot": 1, "id": 5627 , "Count": 1 , "Damage": 32 }, 
    { "Slot": 2, "id": 5627 , "Count": 1 , "Damage": 16 }
  ]
  }), [[null,enderIOTank,null],[bcAdditionsCoilLava,frame,bcAdditionsCoilLava],[odPlateInvar,advGenePowerIO,odPlateInvar]]);
```

**Comment by jaredlll08** [comment]
What crafttweaker version?

**Comment by DerOli82** [comment]
1.7.10 3.0.13

**Comment by jaredlll08** [comment]
Well it is fixed in 3.0.14, but it is 1.10 only

**Comment by jaredlll08** [comment]
Also 1.7.10 is no longer supported
